src/Uno.UWP/Devices/Power/BatteryReport.Android.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.Devices.Power
+{
+	public sealed partial class BatteryReport
+	{
+		public int? ChargeRateInMilliwatts { get; }
+		public int? DesignCapacityInMilliwattHours { get; }
+		public int? FullChargeCapacityInMilliwattHours { get; }
+		public int? RemainingCapacityInMilliwattHours { get; }
+		public System.Power.BatteryStatus Status { get; }
```

Comment:
Consider adding `using System.Power;` so that you can remove the `System.Power.` prefix everywhere.

Reply:
Using full namespace is better (I think), because it clearly differentiate between same name in Android and in UWP. And this is especially true in code like this, then we simply convert between UWP and Android.

Reply:
Can this aliases be `UWP` and `Android`?

Reply:
You can use `UwpBatteryStatus` and `AndroidBatteryStatus` for example. I think I have used similar approach in some other sensor class